docs: ViewChild/ContentChild read property is not a boolean

dtaylor84 opened the pull request:
I still don't entirely understand what the read property is, but it's not a boolean and describing it as "True to..." is misleading.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:


## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

Issue Number: N/A


## What is the new behavior?


## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No


<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

mary-poppins commented:
You can preview fbde505 at https://pr35340-fbde505.ngbuilds.io/.

kara reviewed (dismissed):
The change LGTM, thanks for fixing!

But can you update the commit message to explain what the change is? We don't want "I still don't understand" to be in the public-facing commit history. How about something like:

```
docs(core): fix ViewChild/ContentChild API docs for the `read` property

Previously, the API docs for ViewChild and ContentChild erroneously 
implied that the `read` property was a boolean. Since `read` actually 
accepts a token, this commit updates the docs to reflect that.
```

kara commented:
@dtaylor84 Can you also sign the CLA?

Commit:
docs(core): fix ViewChild/ContentChild API docs for the `read` property

Previously, the API docs for ViewChild and ContentChild erroneously
implied that the `read` property was a boolean. Since `read` actually
accepts a token, this commit updates the docs to reflect that.

mary-poppins commented:
You can preview f6a497c at https://pr35340-f6a497c.ngbuilds.io/.

petebacondarwin reviewed (approved):
Reviewed-for: global-docs-approvers

petebacondarwin commented:
I'm afraid that this documentation issue actually got resolved already in f6a497c87dd2f3f0d8ba5f291afbf76dd68be14c.
